src/map/MapNodeInfo.js

```diff
@@ -74,14 +74,14 @@ function MapNodeInfoMain (props) {
   }, [system]);
 
   return (
-    <Card className='encointer-map-node-info' style={props.style || {}}>{
-      apiState === 'READY'
-        ? <React.Fragment>
-          <Card.Content className='info' onClick={onClickNode}>
-            <Card.Header>{nodeInfo.nodeName} <Icon name='chevron down' /></Card.Header>
-            <Card.Meta>{`${nodeInfo.chain || ''} v${nodeInfo.nodeVersion}`}</Card.Meta>
-          </Card.Content>
-          <Card.Content className='blocks'>
+    <Card className='encointer-map-node-info' style={props.style || {}}>
+      <Card.Content className='info' onClick={onClickNode}>
+        <Card.Header>{nodeName(nodeInfo)} <Icon name='chevron down' /></Card.Header>
+        <Card.Meta>{chainInfo(nodeInfo)}</Card.Meta>
```

Comment:
Always display the dropdown error so that we can switch the network after a connection error